### PR TITLE
Reducing deployment target

### PR DESCRIPTION
**Pull request by tbaranes**
That framework looks really great, but the deployment target is quiet hight for my projects 😬

So, I updated (for carthage and CocoaPods) them to:
- iOS: 8.0
- macOS: 10.0
- tvOS: 9.0 (no need for that one, but I think it's for the best)

Is there any reasons to have higher version? I didn't see any regressions by running the tests / launching the example.

### CODE REVIEW

**Comment by wokalski** [comment]
Thanks! We actually need to make CI work on those versions, too. In order to accomplish that we need to tweak Travis setup. It is currently being changed in #16 so I'd wait until it's merged.

Thanks again. I'll notify you when the other PR is merged.

**Comment by wokalski** [comment]
@tbaranes OK, #16 is merged. Feel free to rebase (or really, it will be easier to just reapply the changes 😜) and update `.travis.yml` (add entries to the [matrix](https://github.com/wokalski/Diff.swift/blob/master/.travis.yml#L21))

**Comment by tbaranes** [comment]
Just rebased, I see that the other PR has already reduced the xcodeproj targets, so it will just for CocoaPods!

Travis up to date too, I added all the os destinations supported by `Diff.swift`. Let met know if there's something missing

**Comment by tbaranes** [comment]
Any ideas why the iOS10+ targets are failing? 🤔

**Comment by wokalski** [comment]
If you view the raw log on Travis you can see that it failed because of a timeout. I saw some discussions about it online but can't find them anymore. @tbaranes.

If you can't find them I'll do my best to fix it over the weekend

**Comment by tonyarnold** [comment]
Yeah, you're looking for this: https://github.com/travis-ci/travis-ci/issues/4725

At the bottom of `.travis.yml`, update it to:

```yaml
after_success:
   - bash <(curl -s https://codecov.io/bash)
   - sleep 5 # Workaround for https://github.com/travis-ci/travis-ci/issues/4725
```

**Comment by tbaranes** [comment]
Right, I already see that kind of issue on others projects. 
I updated .travis using @tonyarnold answer, thanks guys!

File: .travis.yml [hunk 1]
```diff
@@ -51,6 +51,6 @@ script:
     - if [ $RUN_TESTS != "YES" ]; then xcodebuild clean build -project "$PROJECT" -scheme "$SCHEME" -sdk "$SDK" -destination "$DESTINATION" ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO | xcpretty -c; fi
 
     - swift build
-
 after_success:
    - bash <(curl -s https://codecov.io/bash)
+   - sleep 5 # Workaround for https://github.com/travis-ci/travis-ci/issues/4725
```
Comment (tonyarnold) [inline .travis.yml]: Just a small note - please ensure there are newlines at the end of all of your files.

Otherwise, all 👍

Comment (tbaranes) [reply .travis.yml]: Updated 👌

**Comment by tbaranes** [comment]
@wokalski Still one fail due to timeout, can you launch it a second time manually?

**Comment by wokalski** [comment]
Awesome @tbaranes. Thanks a lot!

**Pull request merged**